### ISSUE TITLE
Cannot use with le_node with jest

### ISSUE THREAD

**Issue by zol** [rapid7/le_node#159]
The `codependency` package interferes with jest's require mocking. 

Folks talking about/working on the issue from the `codependency` side here https://github.com/Wizcorp/codependency/pull/14

And on the jest side here https://github.com/facebook/jest/issues/3419

Unfortunately this blocks me from using LogEntries at our company. I can provide more info/help if the above isn't enough to go on.

**Comment by mericsson** [comment]
+1 hitting this too... blocking jest rollout at Mixmax :disappointed:

**Comment by pldin601** [comment]
As a workaround you can use some sort of proxy module, that replace logger with stub in tests:

```javascript
class LoggerStub {
  log(lvl, log) {}
}

export default process.env.NODE_ENV === 'test' ? LoggerStub : require('le_node');
```

and require it your code insead of  `le_node` directly.

**Comment by pldin601** [comment]
Or use jest's __mocks__.

**Comment by mericsson** [comment]
Yep, got around it with by stubbing / mocking it out. Thanks for the suggestions!
